### PR TITLE
create routes for logout login and forgotPassword

### PR DESCRIPTION
**Pull request by AndriyRubanskui**

### CODE REVIEW

File: routes/logout.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const express = require('express');
+
+const router = express.Router();
+router.get('/', (req, res) => {
+  res.send('it is a logout block');
```
Comment (InnaKutsik) [inline routes/logout.js]: pls add TODO

File: routes/login.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+const express = require('express');
+
+const router = express.Router();
+router.get('/', (req, res) => {
+  res.send('it is a login block');
```
Comment (InnaKutsik) [inline routes/login.js]: pls add TODO

File: routes/forgotPassword.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const express = require('express');
+
+const router = express.Router();
+router.get('/', (req, res) => {
+  res.send('it is a forgot password block');
```
Comment (InnaKutsik) [inline routes/forgotPassword.js]: pls add TODO

File: routes/logout.js (hunk 1 above)
Comment (AndriyRubanskui) [reply routes/logout.js]: Ok

File: routes/login.js (hunk 2 above)
Comment (AndriyRubanskui) [reply routes/login.js]: Ok

File: routes/forgotPassword.js (hunk 3 above)
Comment (AndriyRubanskui) [reply routes/forgotPassword.js]: ok

**Pull request merged**